### PR TITLE
Add FileTypeClassifier Caveat

### PR DESCRIPTION
**Pull request by brandenchan**

### CODE REVIEW

**Review by ZanSara — APPROVED** [review]
:+1:

File: docs/latest/pipeline_nodes/file_classifier.mdx [hunk 1]
```diff
@@ -26,6 +26,13 @@ These are the default outgoing edges of the File Classifier:
 | 4             | Docx      |
 | 5             | HTML      |
 
+<div className="max-w-xl bg-yellow-light-theme border-l-8 border-yellow-dark-theme px-6 pt-6 pb-4 my-4 rounded-md dark:bg-yellow-900">
+
+**Note: ** The FileTypeClassifier works best when you pass in one file per `Pipeline.run()` call.
+    If multiple files are passed in to `Pipeline.run()` or `Pipeline.run_batch()`, the FileTypeClassifier currently throws an error if they are not of the same type.
```
Comment (agnieszka-m) [inline docs/latest/pipeline_nodes/file_classifier.mdx:32]: How about:
If you pass multiple files of different format into `Pipeline.run()` or `Pipeline.run_batch()`, FileTypeClassifier returns an error.

what if the files are of one type? is it then OK to pass multiple files to these methods?

Comment (agnieszka-m) [inline docs/latest/pipeline_nodes/file_classifier.mdx]: How about:
If you pass multiple files of different format into Pipeline.run() or Pipeline.run_batch(), FileTypeClassifier returns an error.

what if the files are of one type? is it then OK to pass multiple files to these methods?

**Review by agnieszka-m — CHANGES_REQUESTED** [review]

File: docs/latest/pipeline_nodes/file_classifier.mdx (hunk 1 above)
Comment (brandenchan) [reply docs/latest/pipeline_nodes/file_classifier.mdx]: Yes I believe you can pass in multiple files of the same type but then at that point, I feel like the FileTypeClassifier becomes redundant.

**Review by agnieszka-m — APPROVED** [review]

**Pull request merged**